test/src/ott-analytics.spec.js

```diff
@@ -314,6 +315,40 @@ describe('_sendAnalytics', () => {
     spy.restore();
   });
 
+  it('should stop sending requests after destroy', done => {
+    let numberOfCalls;
+    const checkStopWorking = () => {
+      spy.callCount.should.equal(numberOfCalls);
+      done();
+    };
+    const destroyPlugin = () => {
+      ottAnalytics.destroy();
+      numberOfCalls = spy.callCount;
+      setTimeout(checkStopWorking, config.mediaHitInterval * 2000);
+    };
+    spy = sinon.spy(ottAnalytics, '_sendAnalytics');
+    ottAnalytics._onMediaLoaded();
+    ottAnalytics._onFirstPlay();
+    ottAnalytics._onPlay();
+    setTimeout(destroyPlugin, 1000);
+  });
+
+  it('Interval should be null after reset', () => {
```

Comment:
pay attention that all that appears in the report is this title, so better give more meaningful names.
Also, it should start with `should` -> `it('should....')`
I suggest you copy the `it('should stop sending requests after destroy', done => {` test and create a 
`it('should stop sending requests after reset', done => {`
In the test check for the `_mediaHitInterval`

Reply:
also, see failing test in CI: `should send bookmark concurrency event when server respond with concurrency error`